OrbitCore/LinuxUtils.cpp

```diff
@@ -86,39 +88,52 @@ std::string ExecuteCommand(const char* a_Cmd) {
 //-----------------------------------------------------------------------------
 void ListModules(pid_t a_PID,
                  std::map<uint64_t, std::shared_ptr<Module>>& o_ModuleMap) {
-  std::map<std::string, std::shared_ptr<Module>> modules;
+  std::map<std::string, std::pair<uint64_t, uint64_t>> module_addresses;
```

Comment:
it is not clear what uints here stand for... add a comment (start_address, end_address)  or use a struct with named fields?

Reply:
done